package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "intl-messageformat",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Comment:
I prefer changing minor version instead of patch version for this change.

Reply:
changed